style.css

```diff
@@ -49,4 +178,23 @@ img {
     .gallery-column {
         flex-basis: 90%;
     }
+
+    .search-input {
+        width: 60vw;
+    }
```

Comment:
We can delete those, they are useless. You already have this rule when the screen is `max-width:770px`. Which means it will also apply if the screen is `max-width:700px`.